packages/twenty-server/src/modules/workspace-member/query-hooks/workspace-member-update-one.pre-query.hook.ts

```diff
@@ -33,6 +45,27 @@ export class WorkspaceMemberUpdateOnePreQueryHook
       },
     );
 
+    // TODO: remove this code once we have migrated locale update to userWorkspace update
+    if (payload.data.locale) {
```

Comment:
Vision is to deprecate most of workspaceMember fields (workspaceSchema) to migrate them to userWorkspace